app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt

```diff
@@ -52,15 +52,15 @@ class MainActivity : AppCompatActivity() {
     private lateinit var bindind: ActivityMainBinding
     internal lateinit var mySwitch : Switch
 
-    companion object {
-        var LOCATION = ""
-    }
 
     @RequiresApi(Build.VERSION_CODES.N)
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         bindind = DataBindingUtil.setContentView(this, R.layout.activity_main)
 
+
+
+
```

Comment:
minor: dont leave white spaces. you can also enable auto-format,auto-organize imports before commit in android studio to keep your code clean ;)

Reply:
Okay this is noted!